### ISSUE TITLE
Crash, mod conflict most likely

### ISSUE THREAD

**Issue by beanaboston** [Tommsy64/SmartMovingReloaded#6]
https://paste.ee/p/nexHP

https://modlister.com/beanaboston

Crashes before loading screen even pops up. I have foamfix, betterfps, and optifine (already tried disabling optifine and that didn't work) I'll try with foamfix and betterfps later. I've got some issues with my CT scripts that i've gotta fix and would like to get working on those.

Works fine with the mod on it's own btw. So most likely a mod conflict.

**Comment by SalmonSays** [comment]
I've been checking launch alongside Foamfix, Optifine, and 152 other mods without problems so far to effectively stress test for problems on my branch. It's possible BetterFPS is the culprit, though why it in particular would cause "Required mixin config mixins.smartmoving.core.json requires mixin subsystem version 0.7.8" to rear its head (sort of again except 0.7.8->0.7.5 instead of 0.7.10->0.7.8) is beyond my current understanding of mixin.

TL;DR it's probably worth checking if BetterFPS specifically causes the crash because if not something wacky is going on under the hood.

**Comment by beanaboston** [comment]
Alright. I'll check that out later.

On Sep 3, 2018 12:35 PM, "SalmonSays" <notifications@github.com> wrote:

I've been checking launch alongside Foamfix, Optifine, and 152 other mods
without problems so far to effectively stress test for problems on my
branch. It's possible BetterFPS is the culprit, though why it in particular
would cause "Required mixin config mixins.smartmoving.core.json requires
mixin subsystem version 0.7.8" to rear its head (sort of again except
0.7.8->0.7.5 instead of 0.7.10->0.7.8) is beyond my current understanding
of mixin.

TL;DR it's probably worth checking if BetterFPS specifically causes the
crash because if not something wacky is going on under the hood.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/Tommsy64/SmartMovingReloaded/issues/6#issuecomment-418158180>,
or mute the thread

**Comment by Tommsy64** [comment]
The mod `LagGoggles-FORGE-1.12.2-4.1.jar` is loading its own older version of Mixins before SmartMoving is (See line 471).

Decreasing the required Mixins version to 0.7.5 shouldn't cause any major issues.

**Comment by HighwayStar54** [comment]
> Decreasing the required Mixins version to 0.7.5 shouldn't cause any major issues.

How could I do that? I'm using MultiMCand I have his same problem, is there any conventional way to change the mixins version?

EDIT: Oh and, for me the incompatible mod is Malisis Core.

**Comment by Tommsy64** [comment]
@HighwayStar54 Malisis core [appears to be loading Mixins 0.7.4](https://github.com/Ordinastie/MalisisCore/blob/1.12/build.gradle#L51) before SmartMoving can load 0.7.8.

**Comment by HighwayStar54** [comment]
Ok, I asked the mod author if they could update their mod, is there anything more I could do or does the waiting game start now?

**Comment by Tommsy64** [comment]
I'm not 100% sure, but I think forge examines mods in alphabetical order. Try renaming the `SmartMoving-1.12.2-0.0.4.jar` to something like `aaaaaaaa_SmartMoving-1.12.2-0.0.4.jar`. This _might_ cause SmartMoving's version of Mixins to be loaded first.
